src/webui/www/private/scripts/client.js

```diff
@@ -923,7 +923,9 @@ window.addEventListener("DOMContentLoaded", () => {
             + window.qBittorrent.Client.mainTitle();
 
         $("freeSpaceOnDisk").textContent = "QBT_TR(Free space: %1)QBT_TR[CONTEXT=HttpServer]".replace("%1", window.qBittorrent.Misc.friendlyUnit(serverState.free_space_on_disk));
-        $("DHTNodes").textContent = "QBT_TR(DHT: %1 nodes)QBT_TR[CONTEXT=StatusBar]".replace("%1", serverState.dht_nodes);
+
+        if (window.qBittorrent.Cache.preferences.get().dht)
```

Comment:
It is ok to do it like this or should it be added to serverState?

Reply:
just keep it as is.